server/migrations/20171128031456-create-user.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('User', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      fullname: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING,
+        unique: true
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      role: {
+        type: Sequelize.STRING,
+        defaultValue: 0
```

Comment:
Why do have a default value for the `role` field ?

Reply:
I have changed it already but I thought of using 0 and 1 for differentiating between roles, thats why I put it there